parts/k8s/kubernetesmastervars.t

```diff
@@ -87,6 +87,8 @@
     "apiVersionStorage": "2018-07-01",
     "apiVersionKeyVault": "2018-02-14",
     "apiVersionNetwork": "2018-08-01",
+    "apiVersionManagedIdentity": "2015-08-31-PREVIEW",
```

Comment:
nit: since both are managed identities, can we name them `apiVersionSystemManagedIdentity` and `apiVersionUserManagedIdentity`.